extensions/amp-iframe/1.0/component.js

```diff
@@ -72,9 +72,7 @@ export function BentoIframe({
 
   useEffect(() => {
     const iframe = iframeRef.current;
-    // TODO(36239): Ensure that effects are properly isolated between test runs.
-    // Guarding for iframe truthiness should be enough.
-    if (!iframe?.ownerDocument.defaultView) {
+    if (!iframe) {
```

Comment:
Is this an intended revert or did you not rebase after merging #36241?

Reply:
Intentionally removed it, since this PR fixes the root cause of the issue.